binaries/conda/build_packages.sh

```diff
@@ -4,12 +4,18 @@ set -eou pipefail
 
 PYTHON_VERSIONS="3.6 3.7 3.8"
 PKGS="torchserve torch-model-archiver"
+TORCHSERVE_VERSION=`cat ../../ts/version.txt`
```

Comment:
It's worth adding these version files in this change to start with "0.1.0"

Reply:
@mycpuorg : version.txt file has already been added for both `torchserve` and `torch-model-archiver` as a part of #360 and thus utilizing them in conda's `meta.yaml` as well.

Reply:
The version has been marked as `0.1.1` considering the upcoming release label. 

https://github.com/pytorch/serve/blob/staging_0_1_1/model-archiver/model_archiver/version.txt
https://github.com/pytorch/serve/blob/staging_0_1_1/ts/version.txt